Rollbar/Diagnostics/RuntimeEnvironmentUtility.cs

```diff
@@ -51,5 +52,31 @@ public static string[] GetAssemblyTargetFrameworks(Assembly theAssembly)
 
             return targetFrameworks;
         }
+
+        /// <summary>
+        /// Gets the OS description.
+        /// </summary>
+        /// <returns></returns>
+        public static string GetOSDescription()
```

Comment:
This is being used where just plain "windows" was used before. From reading the docs this string will likely have more information and not just be "windows". I think the backend will be able to handle that and still show the right OS logo but I'm not sure. Something similar broke PHP language detection when we enriched the language value. As long as the items being sent still look the same in the UI then everything is okay.

Reply:
Looks like the platform is just presented as a string:
![image](https://user-images.githubusercontent.com/1753993/42049609-50726828-7abb-11e8-8575-77a2a118bff9.png)
and with the code changes it looks like this now:
![image](https://user-images.githubusercontent.com/1753993/42049710-a102dbec-7abb-11e8-823c-dffa2712d0bf.png)
Is there any other place when a logo is presented instead?

Reply:
In the list of items there are logos sometimes depending on the platform and language. I looked into it and I was wrong, this is okay.

Reply:
In the list of items there are logos sometimes depending on the platform and language. I looked into it and I was wrong, this is okay.

Reply:
@rokob , thanks for double checking it. i'll be merging the PR in a few minutes.

Reply:
@rokob, btw, it looks like the items with request element populated have OS logos:
![image](https://user-images.githubusercontent.com/1753993/42062759-86e8f174-7ae3-11e8-86cc-3c7a0d065041.png)
and the top ones with today's date already use new more detailed OS name/version format. Looks like it works...